CAPE/CAPE/run_cape_offline.cpp

```diff
@@ -122,24 +122,25 @@ int main(int argc, char ** argv){
     float cx_rgb = K_rgb.at<double>(0,2); float cy_rgb = K_rgb.at<double>(1,2);
 
     // Read frame 1 to allocate and get dimension
-    cv::Mat rgb_img, d_img;
+    cv::Mat d_img;
     int width, height;
     stringstream image_path;
     stringstream depth_img_path;
-    stringstream rgb_img_path;
-    rgb_img_path<<string_buff.str()<<"/rgb_0.png";
+    stringstream save_path;
     depth_img_path<<string_buff.str()<<"/depth_0.png";
+    
 
-    rgb_img = cv::imread(rgb_img_path.str(),cv::IMREAD_COLOR);
+    d_img = cv::imread(depth_img_path.str(),cv::IMREAD_ANYDEPTH);
```

Comment:
I removed rgb_img. But authors used rgb_img in order to know `width` and `height`. So instead of rgb I used depth_img in order to know `width` and `height`